golem/task/tasksession.py

```diff
@@ -944,12 +946,12 @@ def _react_to_ack_report_computed_task(self, msg):
         delayed_forcing_msg = message.concents.ForceSubtaskResults(
             ack_report_computed_task=msg,
         )
-        logger.debug('[CONCENT] ForceResults: %s', delayed_forcing_msg)
         ttc_deadline = datetime.datetime.utcfromtimestamp(
             msg.task_to_compute.compute_task_def['deadline']
         )
         svt = msg_helpers.subtask_verification_time(msg.report_computed_task)
         delay = ttc_deadline + svt - datetime.datetime.utcnow()
+        logger.debug('[CONCENT] Delayed ForceResults: %s, delay: %s', delayed_forcing_msg, delay)

```

Comment:
https://github.com/golemfactory/golem-internal/wiki/Logging-guidelines#log-message-structure
```suggestion
         logger.debug('[CONCENT] Delayed ForceResults. msg=%r, delay=%r', delayed_forcing_msg, delay)
```